website/app/styles/doc-components/table-of-contents.scss

```diff
@@ -23,6 +23,10 @@ $doc-toc-item-inset: 8px;
       background-color: var(--doc-color-gray-500);
       content: "";
     }
+    // last one doesn't have the separator
+    &:last-child::after {
+      content: none;
+    }
```

Comment:
This is fine but I usually do something like below to avoid having to override the style:

```
.doc-table-of-contents__item--depth-1 + .doc-table-of-contents__item--depth-1 {
   /* in-between border styles here (using ::before vs. ::after) */
}
```

Reply:
yeah, I found I simpler way to do it using your suggestion, thanks